### DISCUSSION TITLE
possible to add custom emote

### DISCUSSION THREAD

**Discussion by Janisku7** [IEvangelist/profanity-filter#22] [Ideas]
like implemention for like [inworld ](https://inworld.ai) emote as commit could be prompt so filter could show it filtered way still with emotion if it is detected

**Comment by IEvangelist** [comment]
This is an interesting idea. How exactly would it work? If profanity is detected, it would use the original text as a prompt? What's that look like?

**Comment by Janisku7** [comment]
the synthetic generated image with visual presentation and if the filter set like ridiculous for it for too sensitive profanity is detected like the example had "some swear words" could be with one emoji way showing it

**Comment by IEvangelist** [comment]
Ah, ok. So, the idea would be to add a replacement strategy named `ridiculous` and when that's configured, the issue, issue comment, or PR would be updated to add an embedded emote? I'm not sure if that's possible. Do you have any examples of that working anywhere?

**Comment by IEvangelist** [comment]
@Janisku7 - Are you able to embed an emote on this issue, as a comment? I could look at the markdown and try to get a better understanding.

**Comment by IEvangelist** [comment]
Let's convert this into a discussion where we can talk through the details as I'm still unclear of how this might work.
